### PR TITLE
added info about Tetiana

### PR DESCRIPTION
**Pull request by TetianaDiachenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3377,3 +3377,20 @@ What's up, guys?
 
 ## Test
 That my test
+
+# Kottans-frontend
+
+<p> Hello! My name is Tania. I want to be a Frontend developer. I hope that this course helps me to achieve the goal. </p>
```
Comment (A-Ostrovnyy) [inline README.md:3383]: Avoid using HTML tags in markdown. Use [this resource](https://www.markdownguide.org/cheat-sheet/) for updating your changes

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome!
Replace HTML tags with corresponding markdown and push it again)

File: README.md (hunk 1 above)
Comment (TetianaDiachenko) [reply README.md:3535]: Hello,
And now? I tried to fix it and pushed again. 
Thank you for the information about markdown, it was new for me)
If I correct understand, tag <p> we can use in markdown too?

File: README.md [hunk 2]
```diff
@@ -3530,6 +3530,23 @@ I'm Dima. I'm happy to be here. My [repo]https://github.com/bulbasaur-d/kottans-
...
 
+# Kottans-frontend
+
+<p> Hello! My name is Tania. I want to be a Frontend developer. I hope that this course helps me to achieve the goal. </p>
+
+____
+
+## Git Basics
+
+<p> I studied the materials of the course Introduction to Git and GitHub on Coursera.</p>
```
Comment (A-Ostrovnyy) [inline README.md:3541]: `<p>` still HTML tag)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Almost done)
Fix one small issue)

File: README.md (hunk 2 above)
Comment (TetianaDiachenko) [reply README.md]: Ok, and now? It's ok?

**Review by A-Ostrovnyy — APPROVED** [review]
Awesome!
But this repo is only for initial Pull Request, all your impressions about the course you should add to your kottans-frontend repo)

**Comment by TetianaDiachenko** [comment]
Thank you! 
Yes, of course, I created a [kottans-frontend repo](https://github.com/TetianaDiachenko/kottans-frontend) with my reports about the materials, which I studied.

**Pull request merged**